redirect.html

```diff
@@ -3,14 +3,19 @@
   <head>
     <meta charset="utf-8">
     <title>Redirecting to Cliff Effects</title>
-    <link rel="canonical" href="./app"/>
-    <meta http-equiv="refresh" content="0;url='./app'" />
+    <link rel="stylesheet" href="redirect.css">
+    <!-- <link rel="canonical" href="./app"/>
+    <meta http-equiv="refresh" content="0;url='./app'" /> -->
   </head>
   <body>
+    <header>Standard Cliff Effects Header goes here</header>
```

Comment:
I'm not sure it's a great idea to try to do the standard header here. If we're not using React, then we'd have to build this part by hand and would have to change stuff here independently if we changed stuff in that header. It would be a pain to maintain. Maybe we could just do something unique to this page? A link to the github repo or something in case the redirect doesn't work? Or no link. Just not something we'll have to keep matching up by hand.